### PR TITLE
Add lint check for missing newlines at end of tests and fix tests without the newlines

### PR DESCRIPTION
**Pull request by devsnek**

### CODE REVIEW

**Comment by leobalter** [comment]
This is the first time I've had an easy time resolving conflicts using the GH tool.

Thanks for this PR, I appreciate the consistency work.

File: test/built-ins/Function/prototype/toString/line-terminator-normalisation-CR.js [hunk 1]
```diff
@@ -1 +1 @@
...
{
// h
;
// i
;
// j
}
// after

assertToStringOrNativeFunction(f, "function\r// a\rf\r// b\r(\r// c\rx\r// d\r,\r// e\ry\r// f\r)\r// g\r{\r// h\r;\r// i\r;\r// j\r}");
```
Comment (leobalter) [inline test/built-ins/Function/prototype/toString/line-terminator-normalisation-CR.js]: Nice job not messing these up

Comment (leobalter) [reply test/built-ins/Function/prototype/toString/line-terminator-normalisation-CR.js]: Out of curiosity, this file had it intentionally, but it doesn't hurt adding a line feed here for consistency

Comment (devsnek) [reply test/built-ins/Function/prototype/toString/line-terminator-normalisation-CR.js]: do you mean ending it with \r\n instead of just \n?

Comment (leobalter) [reply test/built-ins/Function/prototype/toString/line-terminator-normalisation-CR.js]: From what I see here, it didn't a line ending before, but it does not affect the test contents at all.

I meant it doesn't hurt to have your addition, so it linter will go smoother.

**Pull request merged**